specification/network/resource-manager/Microsoft.Network/stable/2019-11-01/networkManagerCommit.json

```diff
@@ -0,0 +1,243 @@
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "DeploymentsPost": {
+            "$ref": "./examples/NetworkManagerCommitPut.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      },
+      "delete": {
+        "tags": [
+          "NetworkManagerCommits"
+        ],
+        "operationId": "Commits_Delete",
+        "description": "The Delete operation deletes a Managed Network Commit",
+        "responses": {
+          "200": {
+            "description": "Delete successful."
+          },
+          "202": {
+            "description": "Accepted and the operation will complete asynchronously."
+          },
+          "204": {
+            "description": "Request successful. Resource does not exist."
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "ConnectivityConfigurationsDelete": {
+            "$ref": "./examples/NetworkManagerCommitDelete.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "azure-async-operation"
+        }
+      }
+    }
+  },
+  "definitions": {
+    "Commit": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/CommitProperties",
+          "description": "The Commit properties"
+        },
+        "name": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource name."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "Resource type."
+        },
+        "etag": {
+          "readOnly": true,
+          "type": "string",
+          "description": "A unique read-only string that changes whenever the resource is updated."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/SubResource"
+        }
+      ],
+      "description": "Configuration and location lists to commit."
+    },
+    "CommitProperties": {
+      "properties": {
+        "configurationId": {
+          "type": "string",
+          "description": "configuration Id need to deploy."
+        },
+        "location": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Location string"
+        },
```

Comment:
Since the requirement is to allow customers SDP the config, API shoudln't allow 'comitting' to multiple locations at once.
it should be singleton, instead of the array.

Reply:
Commit could go to mulitple regions at once